### PR TITLE
Makes Readmin seperate from reload admins

### PR DESCRIPTION
**Pull request by Supermichael777**
Not as complex as it first seemed

Fixes: #122
image

### CODE REVIEW

File: code/modules/admin/admin_verbs.dm [hunk 1]
```diff
@@ -264,7 +264,7 @@ var/list/admin_verbs_mod = list(
 	/client/proc/cmd_admin_subtle_message,	/*send an message to somebody as a 'voice in their head'*/
 	/client/proc/cmd_admin_xeno_report,  //Allows creation of IC reports by the Queen Mother
 	/proc/release,
-	/datum/admins/proc/viewUnheardAhelps, //Why even have it as a client proc anyway? �\_("/)_/�
+	/datum/admins/proc/viewUnheardAhelps, //Why even have it as a client proc anyway? �\_("/)_/�
```
Comment (ninjanomnom) [inline code/modules/admin/admin_verbs.dm]: I can't believe you'd break the shrug like this

File: code/modules/admin/holder2.dm [hunk 2]
```diff
@@ -88,3 +88,75 @@ you will have to do something like if(client.holder.rights & R_ADMIN) yourself.
 		cdel(holder)
 		holder = null
 	return 1
+
+/client/proc/readmin()
+	if(config.admin_legacy_system)
```
Comment (ninjanomnom) [inline code/modules/admin/holder2.dm]: This entire proc is quite terrible, take a look at how we do it in tg

Comment (Supermichael777) [reply code/modules/admin/holder2.dm]: TG has a different DB backend and admin handler. porting that might take a while™

File: code/modules/admin/admin_verbs.dm (hunk 1 above)
Comment (Supermichael777) [reply code/modules/admin/admin_verbs.dm]: �\_("/)_/� blame atoms lack of UTF8 suport

[the remark this answers is not in the document]
Comment (Supermichael777) [reply code/modules/admin/admin_verbs.dm]: yeah this is kinda fucked

**Comment by Supermichael777** [comment]
If someone wants to step up to bat and do the long haul fix of this feel welcome to it

**Pull request closed without merging**